spec/jobs/index_annotation_job_spec.rb

```diff
@@ -35,6 +36,22 @@
       end
     end
 
+    context 'when no annotation can be found' do
+      let(:exhibit) { FactoryBot.create(:exhibit) }
+
+      before do
+        annotation.delete
+        allow(AnnotationResource).to receive(:new).with(exhibit: nil,
+                                                        annotations: [annotation.to_global_id]).and_return(stub_resource)
+      end
+
+      it 'discards the job' do
+        perform_enqueued_jobs do
+          described_class.perform_later(annotation)
```

Comment:
Is there an assertion happening here?

Reply:
So its a failing test without the change, I guess I can explicitly assert does not raise?

Reply:
I've pushed up the assertion. Nice catch!